src/scss/layout/_navbar.scss

```diff
@@ -40,6 +40,12 @@
         padding-left: add(calc(#{$container-padding-x} / 2), 1.75rem);
         color: inherit;
 
+      &.disabled {
+          color: var(--#{$prefix}gray-300);
+          pointer-events: none;
+          background-color: transparent;
+      }
+
```

Comment:
I would suggest properly aligning these with the rest. It breaks the flow and decreases readability.